src/api/index.js

```diff
@@ -404,3 +404,48 @@ export async function determineInstallNamespace() {
     });
   return response;
 }
+
+export function getTriggerTemplates({ filters = [], namespace } = {}) {
+  let queryParams;
+  if (filters.length) {
+    queryParams = { labelSelector: filters };
+  }
```

Comment:
Since we have this same code snippet copied and pasted for all our `get` functions that use `filters`, should we create a helper function to do this? 😅

Reply:
```
function getQueryParams(filters) {
  if (filters.length) {
    return { labelSelector: filters };
  }
  return "";
}
```

adding